ComponentKit/Core/Scope/CKDetectComponentScopeCollisions.mm

```diff
@@ -36,6 +38,7 @@ void CKDetectComponentScopeCollisions(const CKComponentLayout &layout)
         queue.push(child.layout);
       }
     }
+    [componentBacktrace addObject:[currentLayout.component class]];
```

Comment:
I don't think this works correctly… you need to remove this when you're done processing all of this component's children.